### DISCUSSION TITLE
Can I use GitHub to create a repository for documents?

### DISCUSSION THREAD

**Discussion by tautologias345** [community/community#23872] [New to GitHub]
<p>Example, a ODT document made with LibreOffice Writer, when I edit, I want to make a backup with GitHub, make a public repository, that everybody can edit this document to help me with ideas. Is it possible? Is it safe? Is it adequate?</p>

**Comment by Zerotask** [comment, marked as answer]
<p>Yes, you can do it and yes it is safe.<br>
But mostly we use Markdown files for such cases.</p>
